src/views/FormUser/FormUser.vue

```diff
@@ -0,0 +1,173 @@
+<script lang="ts" setup>
+import { ref, reactive } from 'vue';
+import { z } from 'zod';
+import { useRouter } from "vue-router";
+import Loading from '../../components/Loading.vue';
+
+console.log('start component user')
+
+const router = useRouter();
+
+interface MessageError {
+  name: string;
+  lastName: string;
+  email: string;
+};
+
+interface User {
+  name: string,
+  lastName: string,
+  email: string,
+};
+
+const schema = z.object({
+  lastName: z.string().min(1, 'Please fill in the Last Name.').regex(/^[A-Za-z][A-Za-z\s]*$/, "Only letters are allowed"),
+  email: z.string().email(),
+  name: z.string().min(1, 'Please fill in the name.').regex(/^[A-Za-z][A-Za-z\s]*$/, "Only letters are allowed")
+});
+
+const formData = ref<User>({
+  name: '',
+  lastName: '',
+  email: '',
+});
+
+const errorsDisplay = reactive<MessageError>({
+  name: '',
+  lastName: '',
+  email: ''
+});
+
+let loading = ref(false);
+
+const handleSubmit = async (): Promise<void> => {
+  try {
+    const validatedData = schema.safeParse(formData.value);
+    errorsDisplay.name = ''
+    errorsDisplay.lastName = ''
+    errorsDisplay.email = ''
+    if (!validatedData.success) {
+      const errorsFormated = validatedData.error.format();
+      errorsDisplay.name = errorsFormated.name?._errors[0] || '';
+      errorsDisplay.lastName = errorsFormated.lastName?._errors[0] || '';
+      errorsDisplay.email = errorsFormated.email?._errors[0] || '';
+      return
+    };
+    loading.value = true;
+    console.log('Dados do formulário:', validatedData);
+
+    await new Promise((resolve, reject) => setTimeout(reject, 2000))
+      .then((resolve) => console.log('resolvida', resolve))
+      .catch((error) => console.log('rejeitada', error));
+
+    loading.value = false;
+  } catch (error) {
+    console.error('Erro ao validar os dados:', error);
```

Comment:
You are using English for the schema errors, keep only one language. Obs: check the whole file.

Reply:
done